includes/subscriptions/class-wc-payments-subscription-service.php

```diff
@@ -205,20 +205,14 @@ public function suspend_subscription( WC_Subscription $subscription ) {
 
 	/**
 	 * Reactivates the WCPay subscription when the WC subscription is activated.
-	 * This is done by making a request to server to unset the "cancellation at end of period" value for the WCPay subscription.
+	 * This is done by making a request to server to unset the "pause_collection" value for the WCPay subscription.
 	 *
 	 * @param WC_Subscription $subscription The WC subscription that was activated.
 	 *
 	 * @return void
 	 */
 	public function reactivate_subscription( WC_Subscription $subscription ) {
-		$this->update_subscription(
-			$subscription,
-			[
-				'cancel_at_period_end' => 'false',
```

Comment:
Hey @aprea I think we'll still need to send this `cancel_at_period_end` meta to make sure the Stripe Subscription isn't going to cancel at period end.

This `reactivate_subscription()` function is handling [two activation cases](https://github.com/Automattic/woocommerce-payments/blob/5b4ca8577e33cd4cd0d3825e347b7b983752bdf7/includes/subscriptions/class-wc-payments-subscription-service.php#L109-L110):
 1. Subscription is reactivated during its `pending-cancel` state
 2. Subscription is reactivated after being put `on-hold`

To make sure the subscription in stripe is reactivated we should send both the `cancel_at_period_end` and `pause_collection` params.

Reply:
Oh, gotcha thanks! I was only considering the second scenario 😅

I'll make that change.